lib/ansible/modules/cloud/vmware/vmware_host_sriov.py

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    "metadata_version": "1.1",
+    "status": ["preview"],
+    "supported_by": "community",
+}
+
+DOCUMENTATION = r"""
+---
+module: vmware_host_sriov
+short_description: Manage SR-IOV settings on host
+description:
+- This module can be used to configure, enable/disable SR-IOV functions an ESXi host.
+- module didn't reboot host after changes, but put in output "rebootRequired" key.
+- User can specify an ESXi hostname or Cluster name. In case of cluster name, all ESXi hosts are updated.
+version_added: 2.9
+author:
+- Viktor Tsymbalyuk (@victron)
+notes:
+- Tested on vSphere 6.0
+requirements:
+- python >= 2.6
```

Comment:
```suggestion
- python >= 2.7
```

Pyvmomi requires 2.7

Reply:
I know I need to change for the rest of them.